Source/OxyPlot.SkiaSharp/SkiaRenderContext.cs

```diff
@@ -11,6 +11,7 @@ namespace OxyPlot.SkiaSharp
     using System;
     using System.Collections.Generic;
     using System.Diagnostics;
+    using System.Drawing;
```

Comment:
I don't think this is needed?

Reply:
Ok I just commited these changes. Could it be merged like this? It would be nice if this makes it into a new release so I can switch back to using the nuget package instead of custom built dll. However, there is no rush. Also, I'm happy to head additional feedback if more changes are needed.